### ISSUE TITLE
breathe crashes in parse_definition

### ISSUE THREAD

**Issue by maximilianriemensberger** [michaeljones/breathe#490]
I hit the following error:

```
Exception occurred:
  File "/usr/local/lib/python3.6/dist-packages/breathe/renderer/sphinxrenderer.py", line 60, in parse_definition
    parser = cpp.DefinitionParser(namestr, self)
TypeError: __init__() takes 2 positional arguments but 3 were given
```
Versions

- sphinx 3.0.0
- breathe 4.14.1

With sphinx 2.4.4 and breathe `4.14.1` everything works as expected.

Any ideas what changed with sphinx 3.0.0? Does breathe already support sphinx 3.0.0?

**Comment by vermeeren** [comment]
Fixed in 4.15.0, see https://github.com/michaeljones/breathe/pull/491#issuecomment-610543065.

**Comment by maximilianriemensberger** [comment]
I verified that 4.15.0 fixes it for me as well. Thanks for the quick response.
